lib/galaxy/datatypes/binary.py

```diff
@@ -126,6 +126,34 @@ def sniff(self, filename):
             return False
 
 
+class Gpr (Binary):
```

Comment:
```suggestion
class Gpr(Binary):
```

Reply:
Also, not sure why this cannot also be a subclass of `Tabular`?

Reply:
It causes set_meta function to fail. There is a header name that causes this in the gpr format  as in the first commit comment. When i rename the header and do it in the tabular way it works just fine. But i will no be tempted to change some technology fromat. Multiple packages are using this format. It is not very satisfactory but works.

Reply:
Do you have a traceback for the set_meta fail?
Also, can you provide a link to the commit you mentioned?

Reply:
Check screenshots for the error : 
The error body:
Traceback (most recent call last):
  File "lib/galaxy/jobs/runners/local.py", line 152, in queue_job
    self._finish_or_resubmit_job(job_state, stdout, stderr, exit_code)
  File "lib/galaxy/jobs/runners/__init__.py", line 428, in _finish_or_resubmit_job
    job_state.job_wrapper.finish(stdout, stderr, exit_code, check_output_detected_state=check_output_detected_state)
  File "lib/galaxy/jobs/__init__.py", line 1374, in finish
    dataset.datatype.set_meta(dataset, overwrite=False)
  File "lib/galaxy/datatypes/tabular.py", line 337, in set_meta
    line = dataset_fh.readline()
  File "/opt/galaxy/.venv/lib/python2.7/codecs.py", line 314, in decode
    (result, consumed) = self._buffer_decode(data, self.errors, final)
UnicodeDecodeError: 'utf8' codec can't decode byte 0xb2 in position 384: invalid start byte
![screenshot from 2019-03-01 13-33-40](https://user-images.githubusercontent.com/29343436/53638570-8c2dad80-3c1e-11e9-9bd4-333d1a460cc3.png)
![screenshot from 2019-03-01 13-32-59](https://user-images.githubusercontent.com/29343436/53638571-8c2dad80-3c1e-11e9-8df4-ad3fa4445fa7.png)